README.md

```diff
@@ -1 +1,314 @@
-# signal-searcher
\ No newline at end of file
+# Requirements
+
+**Signal Searcher attempts to find interesting and important signals in MLab's Internet
+measurement data using computers instead of people.**
+
+[Measurement Lab](mlab) is collecting lots and lots of measurement data from all over the Internet. The project is drowning in data, and too frequently has to work too hard to acquire any of data's derivatives: information, knowledge, and wisdom.  In the past, issues were found because there were operator rumors of bad behavior. Depending on intensive investigations by humans spurred by rumor-mill dross is not a sustainable solution. As a process it is too random to get started reliably, and for its execution it depends on a very scarce resource (researcher time).
```

Comment:
In order: 

It is certainly correct that we are drowning in data.  Was the Internet in the US performing better or worse last month than the month before? Did any new peering wars flare up and cause problems? Each of these questions is currently a multi-week research effort for us, and a multi-month effort for not-us. I don't know to what degree our data is useful outside the US, because to find out anything at all is a multi-week research effort. When everything takes many weeks, no research occurs. This is intended to hunt out problematic patterns that can be subsequently displayed by our improved viz system.

Is the data sufficient to support what claims? I feel like this is falling back to the old question we went roundabout for weeks with of "is the data statistically significant", and I kept asking "statistically significant for what?"

I don't understand what the Turkey stuff has to do with this. If you are armed with a question, then using Bocoup's stuff it should be possible to zoom in and get an answer or find out you can't get an answer. Signal Searcher goes at it from the other direction, because Bocoup is producing a visualization system that has more parameters than a human can fully explore, and so we need an automated system to find important and interesting parts of our data.  

When you say "this case", do you mean "Who is the researcher who will be clicking around through signal searcher's results?" or do you mean "Who is looking into the Turkey thing?"  The answer to the first is: once our data is presented in a legible format, I hope that more people will be more interested in it.  The answer to the second is: I don't know. It's currently too hard to do on whim, and that's why we hired Bocoup.